CONTRIBUTING.md

```diff
@@ -275,6 +275,26 @@ By giving feedback on this work (and related supporting work), you can help us m
 Finally, if nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
 As discussed in [*How we're organized*](#how-were-organized), this role only requires good faith and a basic understanding of our development process.
 
+### How to adopt pull requests
+
+Occasionally authors of pull requests get busy or become unresponsive.
```

Comment:
this is still putting all the fault on PR authors

Reply:
Sorry, we were sure we had committed the changes, now it's there.